### PR TITLE
add comments, change user_id to username on db and and profileImage on db

### PR DESCRIPTION
**Pull request by ruben1**

### CODE REVIEW

**Comment by ruben1** [comment]
so there's 2 test failing because they are looking for a user_id key.

File: server/api/screenshot/screenshotModel.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ var db = require('../../db/index.js');
 var screenshotsSchema = mongoose.Schema({
   url: { type: String, required: true },
   originalImage: { type: String, required: true },
-  annotatedImage: { type: String, required: true },
-  user_id: { type: String, required: true },
+  annotatedImage: { type: String, required: true }, //--> annotation functionality to be implemented on the front-end, see wiki for more info about libraries suggested
```
Comment (AndrewSouthpaw) [inline server/api/screenshot/screenshotModel.js]: Add as a separate comment line above

**Comment by AndrewSouthpaw** [comment]
Comments should have a space after `//` (per style guide).

I'll update the tests. Looks good! Did you verify `user_id` is not referenced anywhere in the client?

**Comment by ruben1** [comment]
The client sends the user through the API endpoint so there's no problem with that. It's still something like api/user/name_of_the_user

**Pull request merged**